deploy/global.yaml

```diff
@@ -34,72 +29,125 @@ grafana:
     create: false
     pspEnabled: false
 
-# TODO: add resource requests/limits and tolerations for all components here
+jaeger:
+  allInOne:
+    resources:
+      requests:
+        cpu: 50m
+      limits:
+        memory: 350Mi
+
+default:
+  # Run all demo components on a separate node pool
+  schedulingRules:
+    nodeSelector:
+      nodepool.sentry.io/name: otel-demo
+    tolerations:
+      - key: "nodepool-dedicated"
+        operator: "Equal"
+        value: "otel-demo"
+        effect: "NoSchedule"
+
+# Demo components
 components:
   adService:
     resources:
       requests:
         cpu: 50m
+    schedulingRules:
+      # Here and below: this is a workaround for an existing issue in the Helm chart (to be fixed).
+      # The final values for nodeSelector and tolerations will still be takend from "default".
+      nodeSelector: {}
```

Comment:
Those will go away when the upstream chart is fixed (I have a fix in the works).